docs/topics/db/aggregation.txt

```diff
@@ -323,6 +323,17 @@ title that starts with "Django" using the query::
 
     >>> Book.objects.filter(name__startswith="Django").aggregate(Avg('price'))
 
+When filtering against :ref:`multiple values <spanning-multi-valued-relationships>`,
```

Comment:
Yes, you also need to that reference above the header, otherwise the link won't work (see the docs build error).